agc/asm.c

```diff
@@ -0,0 +1,399 @@
+/*
```

Comment:
Yep. I am aware of the logic dup between asm and anal. And that will be probably merged sometime in the future. But i think its good to have this logic splitted, but it should be merged to avoid decoding twice every instruction. This will be a 2x speedup in disasm for example. But dont worry too nuch for now because its not really a problem and can be fixed later

> On 27 Jul 2016, at 01:39, ibabushkin <notifications@github.com> wrote:
> 
> In agc/asm.c:
> 
> > @@ -0,0 +1,399 @@
> > +/*
> The anal and asm plugins have a lot of shared code that would be hard to interlink properly without a lot of fuss with the build system which I wanted to avoid. This way, all code is in one place. Also, I suppose if someone installs one of the plugins, that person probably wants both, as this target is pretty specific. I asked on IRC when I was deciding this bit and got told by @Maijin that this is an acceptable solution.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>